### PR TITLE
[Backend] Print Selected Radio Button to Console

### PR DESCRIPTION
**Pull request by sghoshjr**
Prints the Selected Radio Button to Console in JSON String Format as {Machine}:{Machine1}, {status}:{print}

### CODE REVIEW

**Comment by sghoshjr** [comment]
@vicennial Check the Diff and Format of console Print

File: website/views/index.html [hunk 1]
```diff
@@ -32,6 +32,25 @@
...
+            data: JSON.stringify(data),
+                contentType: 'application/json',
+                        url: 'SendState',            
+                        success: function(data) {
+                            console.log('success');
+                            console.log(JSON.stringify(data));
+                        }
+                    });
+    }
+    </script>
```
Comment (vicennial) [inline website/views/index.html]: Use this instead: 
```
  <script>
  $(document).ready(function () {
    $(".radioButton").on("change", function () {
      data.status=this.innerText;
      //Add remaining code
    });
  });
  </script>
```  
With this, you dont have to write the tag 'onclick' for each radio button.

**Review by vicennial — CHANGES_REQUESTED** [review]

**Comment by sghoshjr** [comment]
@vicennial Updated PR

**Review by vicennial — APPROVED** [review]

File: website/views/index.html [hunk 2]
```diff
@@ -32,6 +32,29 @@
...
+    <script type="text/javascript" src="js/jquery.min.js"></script>
+
+    <!-- Print the Selected Status to Console -->
+    <script type="text/javascript">
+      $(document).ready(function () {
+         $(".radioButton").on("change", function () {
+          var data = {};
+          data.machine = "Machine1";
+          data.status = this.innerText;
+
```
Comment (vicennial) [inline website/views/index.html]: Add parent mode too.
Use this:
```
data.parent_mode=$(this).parents()[2].firstChild.nextSibling.innerText;
````

**Review by vicennial — CHANGES_REQUESTED** [review]

**Review by vicennial — APPROVED** [review]

**Pull request merged**